### PR TITLE
Cards restyle

### PR DESCRIPTION
**Pull request by saulyz**
This is Work In Progress push

### CODE REVIEW

File: src/components/animal/AnimalAvatar.tsx [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+import IconGuineapig from './IconGuineapig';
+import IconRabbit from './IconRabbit';
+import IconReptile from './IconReptile';
+
+export default function AnimalAvatar({ species, gender }: AnimalAvatarProps) {
+    if (!gender) {
+        gender = {} as Gender;
+    }
+    if (!species) {
+        species = { id: '0', value: 'Undefined' } as Species;
```
Comment (rikius) [inline src/components/animal/AnimalAvatar.tsx:19]: this is not needed.

**Review by rikius — CHANGES_REQUESTED** [review]

File: src/components/animal/AnimalAvatar.tsx [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+}
+
+const useStyles = makeStyles({
+    dynamicAvatar: (props: AvatarStyleProps) => ({
+        color: props.color,
+        backgroundColor: props.backgroundColor,
+    }),
+});
+
+type OptionalGender = Gender | undefined | null;
```
Comment (PPrakapavicius) [inline src/components/animal/AnimalAvatar.tsx:32]: Why are these needed? Can't you just use Gender | undefined | null inside the interface?
